### ISSUE TITLE
[Zul Farrak] Shadowcaster casting Dismounting Blast

### ISSUE THREAD

**Issue by Massie323** [Atlantiss/NetherwingBugtracker#8365]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 
TBC Wowhead lists "[Dismounting Blast](https://www.wowhead.com/tbc/spell=18396/dismounting-blast)" as an ability for [Sandfury Shadowcaster](https://www.wowhead.com/tbc/npc=5648/sandfury-shadowcaster), however this is wrong as the ZF dismounting adjustments were introduced in patch 3.1.0.
Due to the lack of vanilla TBC footage of mounted players running around the dungeon, I'll have to prove and explain it in an indirect manner utilizing WOTLK:
Inside Zul'farrak, by patch 3.3.5a there are 2 mobs that supposedly have a dismounting spell according to Wowhead:

   - [Dismounting Blast](https://www.wowhead.com/tbc/spell=18396/dismounting-blast) used by [Sandfury Shadowcaster](https://www.wowhead.com/tbc/npc=5648/sandfury-shadowcaster); 
   - [Dismounting Shot](https://www.wowhead.com/tbc/spell=18395/dismounting-shot) used by [Sandfury Shadowhunter](https://www.wowhead.com/tbc/npc=7246/sandfury-shadowhunter);  added in undocumented change in patch 3.1.0 according to comment section

Here are 2 ZF AoE pull videos during Wotlk before patch 3.1.0, so before the dismounting spells were introduced:

https://www.youtube.com/watch?v=BMlikVPFTBw
https://www.youtube.com/watch?v=XTdiQzRuozQ

Note that while WOTLK & TBC Wowhead lists both "[Dismounting Blast](https://www.wowhead.com/tbc/spell=18396/dismounting-blast)" and "[Dismounting Shot](https://www.wowhead.com/tbc/spell=18395/dismounting-shot)" as being ingame, the spell is completely absent in both videos, and both players have no issues pulling the entire dungeon while mounted. On a side note, the spells were also completely absent for the entirety of Wow Classic.
Additionally, when reviewing the comments in one of the videos and on Wowhead, talks about "getting dismounted" only begin with patch 3.1.0.

Even cross-referencing multiple other TBC Databases shows the absence of "[Dismounting Blast](https://www.wowhead.com/tbc/spell=18396/dismounting-blast)" for [Shadowcasters](https://www.wowhead.com/tbc/npc=5648/sandfury-shadowcaster), which explains why many Paladins & Mages don't remember being dismounted by the first packs in ZF on other servers:
http://tbc.cavernoftime.com/npc=5648#abilities
https://wowclassicdb.com/tbc/npc/5648
https://db.moonwell.su/?spell=18396
https://70.wowfan.net/en/?npc=5648#abilities

TL;DR:
Wowhead lists dismounting spell that was never used by the NPCs before patch 3.1.0, neither on Blizzard servers, nor on other private servers.

**Current behaviour**:
[Sandfury Shadowcaster](https://www.wowhead.com/tbc/npc=5648/sandfury-shadowcaster) using "[Dismounting Blast](https://www.wowhead.com/tbc/spell=18396/dismounting-blast)".

**Expected behaviour**: 
[Sandfury Shadowcaster](https://www.wowhead.com/tbc/npc=5648/sandfury-shadowcaster) should not use "[Dismounting Blast](https://www.wowhead.com/tbc/spell=18396/dismounting-blast)".

**Server Revision**: 
3757

**Comment by Wolffenstein** [comment]
![image](https://user-images.githubusercontent.com/1576644/233378896-96537482-949e-40e7-9877-f209639dd452.png)
